### DISCUSSION TITLE
Welcome to einverne.github.io Discussions!

### DISCUSSION THREAD

**Discussion by einverne** [einverne/einverne.github.io#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by esmusssein33** [comment]
不好意思，我是反派的效愚，是这样的，可以请你把下载节目当中付费的那一部分拿掉吗？

**Reply by einverne** [reply]
已经通过豆瓣私信解决。
